README.md

```diff
@@ -1 +1,53 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project was created as part of the Code Reviewers training job application at Microverse.org
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] The screenshot link isn't working. Please fix this issue before submitting for a new code review